README.md

```diff
@@ -64,11 +64,11 @@ directory and the Nektar++ submodule (respectively). You can leave
 this environment at any time by running `deactivate`.
 
 `spack install` will build two copies of NESO: one with
-GCC/hipSYCL/FFTW3 and one with Intel's OneAPI/DPC++/MKL. These
+GCC/adaptivecpp/FFTW3 and one with Intel's OneAPI/DPC++/MKL. These
```

Comment:
```suggestion
GCC/AdaptiveCpp and one with Intel's OneAPI/DPC++. These
```
Update for capitalisation + remove now defunct reference to ffts.

Reply:
Done